Ecosystem Partners/Marketplace Guides/getting-started-with-fortycloud-appliance.md

```diff
@@ -0,0 +1,160 @@
+  </div>
+  <div style="float:right;width:45%;margin-right:1em;">
+    *Email #2: Appliance deploy complete*
+    <img src="../../images/bpimager/appliance_deploy_complete.png" style="border:0;width:100%;">
+  </div>
+  <br style="clear:both;">
+
+  Wait for the second email indicating your appliance is ready for use before attempting to access the resource.
+
+8. **Enabling Access to Your Appliance**
+
+  Your appliance will be automatically configured with a public IP address accessible from the following ports:
+
+  * HTTPS (443)
+  * SSH (22)
+  * 500/UDP, 1701/UDP, 4500/UDP
+
+9. **Install a New Gateway**
+
+  * In order to proceed with the following steps you will need an active FortyCloud account.  
+    You can request a new account free of charge on the FortyCloud website: http://campaign.fortycloud.com/30-day-free-trial/
+  * Login to the FortyCloud Web Admin Console to retrieve a new Gateway registration token.
+  * Press on the “Setup a new Gateway” icon
+    <img src="../../images/fortycloud/setup_new_gateway.png" style="">
+  * **Setup a New Gateway** Wizard opens. Make Sure **CenturyLink Cloud** is selected and copy the *FortyCloud Account Registration Token* 
+    (the blue highlighted text)
+    <img src="../../images/fortycloud/setup_new_gateway2.png" style="">
+  * SSH to your server using your password (ssh root@[Public IP]) once prompted enter your password, once you are logged in the following 
+    screen will be prompted
+    <img src="../../images/fortycloud/setup_new_gateway3.png" style="">
+  * Type the following command inside your SSH console
+    ```
+	> sudo fccli register –t <Paste “FortyCloud Account Registration Token”>
+	```
+	You should see the following
+    <img src="../../images/fortycloud/setup_new_gateway4.png" style="">
+  * Within a few minutes your new FortyCloud Gateway will appear on the Web Admin Console dashboard (as seen below)
+    <img src="../../images/fortycloud/setup_new_gateway5.png" style="">
+
+  You can now proceed with the Gateway configuration. 'How to' tutorials are available via the Support Center link located on the top right corner. 
+
+
+### Pricing
+
+The costs listed above in the above steps are for the infrastructure only.
+
+After deploying this Blueprint, you may secure entitlements to the technology using the following steps:
+
+ * Email: sales@fortycloud.com
+ * Visit our pricing plan page: http://fortycloud.com/pricing/ and take advantage of FortyCloud's 30-day free trial offer
+
+
+### Frequently Asked Questions
+
+**Where do I obtain my license?**
+
+In order to create a new license visit our new account request page: http://campaign.fortycloud.com/30-day-free-trial/
+
+
+**Who should I contact for support?**
+
+* For issues related to deploying CloudBolt email centurylink@fortycloud.com.
+* For issues related to cloud infrastructure, please open a ticket using the [CenturyLink Cloud Support Process](../../Support/how-do-i-report-a-support-issue.md).
```

Comment:
@keithresar  Please update your image syntax.  This article has "contentIsHTML": false